Source/WebKit/WebProcess/WebPage/EventDispatcher.cpp

```diff
@@ -87,7 +87,6 @@ void EventDispatcher::addScrollingTreeForPage(WebPage* webPage)
 void EventDispatcher::removeScrollingTreeForPage(WebPage* webPage)
 {
     Locker locker { m_scrollingTreesLock };
-    ASSERT(m_scrollingTrees.contains(webPage->identifier()));
```

Comment:
When UI side compositing is in use, if this assertion isn't valid, can we avoid calling removeScrollingTreeForPage entirely? And avoid touching m_scrollingTreesLock.